packages/manager/modules/pci/src/index.js

```diff
@@ -24,6 +24,7 @@ import '@ovh-ux/ui-kit/dist/css/oui.css';
 import 'ovh-ui-kit-bs/dist/css/oui-bs3.css';
 import 'font-awesome/css/font-awesome.css';
 import 'ovh-common-style/dist/ovh-common-style.css';
+import 'd3';
```

Comment:
Very good initiative 👍 . Instead of importing it in PCI module, can you import only in module/route it is used and lazyload it?

Reply:
I'm not sure that `d3` dependency is needed for following module `@ovh-ux/manager-pci`.

However the given module `@ovh-ux/ng-ovh-cloud-universe-components` has defined `d3` as a Peer Dependency meaning that it must be available on the parent application.

IMO, wherever `@ovh-ux/ng-ovh-cloud-universe-components` is installed, the parent application need to have `d3` from the dependency list.

Reply:
i think you're right, some component from cloud-universe-components are only used once, maybe simply move those components we allow removing d3 from manager-dedicated build

closing this one, i'll reopen a new PR